### DISCUSSION TITLE
Optimized USER.cfg for Version 3.17.1-LIVE.8073467

### DISCUSSION THREAD

**Discussion by Isaard** [Isaard/Optimized-USER.cfg#1] [General]
First Release
Optimized USER.cfg for Version 3.17.1-LIVE.8073467 and readme file with instructions

<hr /><em>This discussion was created from the release <a href='https://github.com/Isaard/Optimized-USER.cfg/releases/tag/v1'>Optimized USER.cfg for Version 3.17.1-LIVE.8073467</a>.</em>

**Comment by scottyloveless** [comment]
What kind of results are you seeing with this user.cfg vs not using it? I'm curious to try it out. Also, you set some values to high, is that based on your system or is everything in this cfg the optimal value?

**Reply by infinitywaves** [reply]
I have a file with about 300 lines of different improvements but RSI decided to enlarge IB Blacklist and nothing wants to work. The game omits the knowledge of user.cfg entries. it's pathetic, they won't do the optimization themselves and they won't let others do it for them. Historically in 3.16 it worked great, often the increase in FPS was at the level of 2x to 3x is a huge difference.

**Comment by infinitywaves** [comment]
Hey, that would look good, but game.log returns "- IGNORING" for most of these entries. How did you go that it works for you?

**Comment by styvynator** [comment]
Worked like a charm for me! I followed your instructions and went up from sub 20 to 40-60FPS. Thanks!
